client/build/Dockerfile

```diff
@@ -7,4 +7,5 @@ RUN apk update && apk upgrade
 RUN apk add git
 
 COPY . /usr/src/client/
+
 RUN npm install
```

Comment:
As we're running the `npm install` command in the specific service with the `command` in docker-compose file, we don't actually need this to be here, right?